en/Plugins/Canvas.md

```diff
@@ -0,0 +1,229 @@
+Canvas is a tool for visual note-taking. Lay out notes and other resources on an infinite canvas and connect them with lines.
+
+Visual note-taking allows you to use visual aspects, such as size and position, to make sense of your notes. Connect notes with lines and group related notes together to better understand the relationship between them.
+
+> [!info] Only for Obsidian desktop
+> Canvas is only available in Obsidian for desktop.
+
+## Create a new canvas
+
+To start using Canvas, you first need to create a file to hold your canvas. You can create a new canvas using the following methods.
+
+**Ribbon:**
+
+- In the ribbon on the left, select **Create new canvas** (four rectangles icon) to create a canvas in the same folder as the active file.
+
+**Command palette:**
+
+1. Open the [[Command palette]].
+1. Select **Canvas: Create new canvas** to create a canvas in the same folder as the active file.
+
+**File explorer:**
+
+- In the [[File explorer]], right-click the folder you want to create the canvas in.
+- Select **New canvas**.
+
+> [!note]  The .canvas file extension
+> Obsidian stores the configuration for each canvas in a custom JSON format in a file with the `.canvas` extension.
+
+## Adding cards
+
+You can drag files into your canvas from Obsidian or from other applications. For example, Markdown files, images, audio, PDFs, or even unrecognized file types.
+
+### Add text cards
+
+You can add text-only cards that don't reference a file.
```

Comment:
A few small points I think we should include here.

- Text-only cards support all the same markdown options as a file. Including `[[links]]` and even plugin code blocks.
- Currently, links in text-only cards are not indexed by Obsidian, so they won't show up in backlinks. To do that, you'll need to convert the card to a file.
-